src/sentry/integrations/github_enterprise/integration.py

```diff
@@ -311,8 +311,10 @@ def get_installation_info(self, installation_data, access_token, installation_id
 
         resp = session.get(
             "https://{}/api/v3/user/installations".format(installation_data["url"]),
-            params={"access_token": access_token},
-            headers={"Accept": "application/vnd.github.machine-man-preview+json"},
+            headers={
+                "Accept": "application/vnd.github.machine-man-preview+json",
+                "Authorization": "token {}".format(access_token),
```

Comment:
Pretty sure this is what the linter wants.

```suggestion
                f"Authorization": "token {access_token}",
```